src/test/java/seedu/address/logic/parser/SetCommandParserTest.java

```diff
@@ -135,59 +135,100 @@ public void parse_someFieldsSpecified_success() {
     }
 
     @Test
-    public void parse_oneFieldSpecified_success() {
-        // name
+    public void parse_onlyNameSpecified_success() {
         String userInput = NAME_DESC_AMY;
-        SetPersonDescriptor descriptor = new SetPersonDescriptorBuilder().withName(VALID_NAME_AMY).build();
+        SetPersonDescriptor descriptor = new SetPersonDescriptorBuilder()
+                .withName(VALID_NAME_AMY)
+                .build();
         SetCommand expectedCommand = new SetCommand(descriptor);
         assertParseSuccess(parser, userInput, expectedCommand);
+    }
 
+    @Test
+    public void parse_onlyAddressSpecified_success() {
         // address
```

Comment:
Perhaps we can remove these meaningless comments since the method name already specified enough.